### DISCUSSION TITLE
Mahapp Demo Source code

### DISCUSSION THREAD

**Discussion by Fl3W** [MahApps/MahApps.Metro#4201] [Q&A] [answered]
I want to download the source code of the demo application because it would be nice to have a working example. Maybe I'm just blind, but I can't find it anywhere.

**Comment by Fl3W** [comment, marked as answer]
Ok, ive found it. I had to open the project src folder and open the project...

**Reply by batzen** [reply]
@Fl3W FYI https://github.com/MahApps/MahApps.Metro/pull/4203
Hope that helps others in the future.
